edx-platform/pearson-theme/lms/templates/header/navbar-authenticated.html

```diff
@@ -44,12 +45,9 @@
         </div>
       % endif
     % endif
-    % if show_explore_courses:
+    % if show_catalogue_link:
       <div class="mobile-nav-item hidden-mobile nav-item nav-tab">
-          <a class="tab-nav-link discover-new-link" href="${marketing_link('COURSES')}"
-             aria-current="${'page' if '/courses' in request.path else 'false'}">
-             ${_('Discover New')}
-          </a>
+        <a class="tab-nav-link" href="${marketing_link('COURSES')}">${_('Discover New')}</a>
```

Comment:
Why was 'aria-current' removed? @amalbas

Reply:
@Squirrel18 sorry, was added in other pr, not in the original so I didn't see it. I fixed it.